schemas/spec-v1.schema.json

```diff
@@ -31,6 +31,7 @@
       "type": "string",
       "title": "Task description",
       "enum": [
+        "3D Visualisation",
```

Comment:
Not sure. Is this something for the Language Resource Switchboard? I would like to hear from others as well

Reply:
This is meant for data repositories with 3d scans of historical artefacts. Not related to language but social sciences.